numpy/core/code_generators/ufunc_docstrings.py

```diff
@@ -4232,3 +4232,36 @@ def add_newdoc(place, name, doc):
     array([ 0, 20, 20, 60, 20, 20])
 
     """)
+
+add_newdoc('numpy.core.umath', 'bit_count',
+    """
+    Computes the number of 1-bits in ``x``.
+    Analogous to the builtin `int.bit_count` or ``popcount`` in C++.
+
+    Parameters
+    ----------
+    x : array_like, unsigned int
+        Input array.
+    $PARAMS
+
+    Returns
+    -------
+    y : ndarray
+        The corresponding number of 1-bits in the input.
+        $OUT_SCALAR_1
+
+    References
+    ----------
+    .. [1] https://stackoverflow.com/a/109025/5671364
+
+    .. [2] Wikipedia, "Hamming weight",
+           https://en.wikipedia.org/wiki/Hamming_weight
+
+    Examples
+    --------
+    >>> np.bit_count(np.uint64(1023))
```

Comment:
```suggestion
    >>> np.bit_count(1023)
```
I suppose the explicit casting to an unsigned integer type is redundant now, no?

Reply:
Oh yes, I missed this. Thanks